client/my-sites/importer/importer-header.jsx

```diff
@@ -19,6 +20,8 @@ import { cancelImport, resetImport, startImport } from 'lib/importer/actions';
 import { connectDispatcher } from './dispatcher-converter';
 import SiteImporterPlaceholderLogo from './site-importer/placeholder-logo';
 
+import { recordTracksEvent } from 'state/analytics/actions';
```

Comment:
Nit: Let's nudge this up one line before we merge :)

Reply:
Updated!